src/components/Radio/styles.ts

```diff
@@ -131,7 +134,7 @@ export const StyledInput = styled.input`
     },
   }: ThemeProp) => xxs};
 
-  ${({ theme }: ThemeProp) =>
+  ${({ theme, disabled }: StyledLabelProps) =>
```

Comment:
Is this the right type? Should it be `ThemeProp & DisabledProp` (same applies for the Checkbox change actuall)